code/style.css

```diff
@@ -2,6 +2,298 @@
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
+
+html {
+  Font-Family: 'Playfair Display', Serif;
+  }
+  
+body  {
+  background-color: #FAF0E6;
+  margin: 0;
+}
+    
+.hero-img {
+  background-image: url(./img/coffe_extreme.jpg);
+  height:350px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover; 
+}
+    
+  
+.hero-img h1 {
+  position: relative;
+  text-align: center;
+  font-size: 35px;
+  color: white;
+  margin:0;
+}
+  
+  /*WE ARE TEXT */
+  
+.weare {
+  display: flex;
+  text-align: center;
+  margin-top: 5px;
+  color: black;
+  font-style: italic;
+  font-size: 10px;
+  text-decoration: underline solid white; 
+}
+  
+  /*sign up text */
+  
+.sign-up {
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  margin-top:30px;
+}
+  
+    /*Container main content*/
+  
+.container{
+  display: inline-flex;
+  flex-direction: column;
+  margin-left:5px;
+  margin-bottom: 30px;
+  margin-top: 30px;
+}
```

Comment:
This isn't centered and I think you should figure out a way to do that because you'll need it for upcoming projects. Play around with the width and margin maybe?